docs/images.md

```diff
@@ -0,0 +1,279 @@
+---
+title: image
+weight: 80
+description: >
+  Handling gadget's OCI images.
+---
+
+> ⚠️ This command is experimental and could change without prior notification. Check the installation guide to enable [experimental features](install.md#experimental-features).
+
+Gadgets in Inspektor Gadget are packaged as OCI images. This document describes the different
+commands available to interact with those images and with OCI registries.
+
+If you're looking to run a gadget, check the [run](./gadgets/run.md) command.
+
+### Authentication
+
+The following to commands are used to login or logout against a container registry.
+
+### `login`
+
+Login to a container registry.
+
+```bash
+$ sudo ig login -h
+Login to a container registry on a specified server.
+
+Usage:
+  ig login [command options] REGISTRY [flags]
+
+Flags:
+      --authfile string   path of the authentication file. Use REGISTRY_AUTH_FILE environment variable to override
+      --cert-dir string   use certificates at the specified path to access the registry
+      --get-login         Return the current login user for the registry
+  -h, --help              help for login
+  -p, --password string   Password for registry
+      --password-stdin    Take the password from stdin
+  -u, --username string   Username for registry
+  -v, --verbose           Write more detailed information to stdout
+
+```
+
+```bash
+$ sudo ig login ghcr.io -u mauriciovasquezbernal
+INFO[0000] Experimental features enabled
+Password:
+Login Succeeded!
+
+$ sudo ig login ghcr.io --get-login
+INFO[0000] Experimental features enabled
+mauriciovasquezbernal
+```
+
+### `logout`
+
+Logout of a container registry.
+
+```bash
+$ sudo ig logout -h
+INFO[0000] Experimental features enabled
+Logout of a container registry on a specified server.
+
+Usage:
+  ig logout [command options] REGISTRY [flags]
+
+Flags:
+  -a, --all               Remove the cached credentials for all registries in the auth file
+      --authfile string   path of the authentication file. Use REGISTRY_AUTH_FILE environment variable to override
+  -h, --help              help for logout
+
+```
+
+```bash
+$ sudo ig logout ghcr.io
+INFO[0000] Experimental features enabled
+Removed login credentials for ghcr.io
+
+$ sudo ig login ghcr.io --get-login
+INFO[0000] Experimental features enabled
+Error: not logged into ghcr.io
+```
+
+### `--authfile`
```

Comment:
We should mention/link this subchapter in `login` and `logout`.

As a user reading a doc i would see the above `login` and `logout` headers. After that I wouldn't probably look further to see details regarding the commands

Reply:
I changed the documentation order to have that information before.